.github/workflows/release.yml

```diff
@@ -544,3 +507,50 @@ jobs:
         asset_path: ${{ matrix.chain }}-srtool-digest.json
         asset_name: ${{ matrix.chain }}-srtool-digest.json
         asset_content_type: application/json
+
+  upload-evm-tracing-artifacts:
+    needs: publish-release-draft
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download pre-built collator binary for evm tracing
+      uses: actions/download-artifact@v3
+      with:
+        name: astar-evm-tracing-ubuntu-latest-x86_64-unknown-linux-gnu
+        path: evm-tracing-artifacts
+    
+    - name: Make evm tracing binary executable and tar gzip
+      run: |
+        cd evm-tracing-artifacts
+        chmod +x astar-collator
+
+    - name: Download evm tracing runtime
+      uses: actions/download-artifact@v3
+      with:
+        name: astar-evm-tracing-runtime
+        path: evm-tracing-artifacts
+
+    - name: Download evm tracing runtime
+      uses: actions/download-artifact@v3
+      with:
+        name: shiden-evm-tracing-runtime
+        path: evm-tracing-artifacts
+    
+    - name: Download evm tracing runtime
+      uses: actions/download-artifact@v3
+      with:
+        name: shibuya-evm-tracing-runtime
+        path: evm-tracing-artifacts
+
+    - name: Compress folder
+      run: |
+        tar zcvf evm-tracing-artifacts.tar.gz evm-tracing-artifacts
```

Comment:
Can you add version in the tar file name?

Reply:
Added version in the name